README.md

```diff
@@ -96,7 +95,38 @@ After adding the line and saving the file, don't forget to source the file
 
     source ~/.bash_profile
 
-#### 3. Running the CMR rake tasks
+#### 3. Setting up local redis for CMR
+CMR comes with redis in the jar, but it is not compiled to run on Macs.  If you need to run the CMR on a Mac, download it from
```

Comment:
I don't think I've ever installed Redis for cmr, is needed?

Reply:
I think that there are certain things that might not run properly if you don't install it. I forget what they are, Ryan probably knows more.

Reply:
I did not compile a list of things that will not run properly without redis, but we recently had a discussion where Charles, John, and William could all reliably reproduce a series of test failures and I could not (I run redis with my uberjar now).  When they added redis, the test failures stopped.  I believe that full functionality of the uberjar is now dependent on it (circa December-ish).

Reply:
Ah that might be why some of my local builds have failed in the past.   Thanks!

Reply:
this is one off the drivers for Docker